### ISSUE TITLE
Issue matching group names (with special characters?)

### ISSUE THREAD

**Issue by rjr162** [gomorpheus/morpheus-cli#12]
We have some role and group names that match, along the format of:

umg/up.5i-some-values-here

When we use to run

morpheus roles update-group-access umg/up.5i-some-values-here umg/up.5i-some-values-here full

morpheus-cli worked fine and everyone was happy

Now when we run:
morpheus roles update-group-access umg/up.5i-some-values-here umg/up.5i-some-values-here full -d

We get:
DRY RUN
Group not found by name 

But if I do:
groups list --wrap --all-fields --search 5i-some

I get:

Morpheus Groups | phrase: 5i-some
==================

ID | NAME                   | LOCATION | CLOUD_COUNT | SERVER_COUNT
---|------------------------|----------|-------------|-------------
4  | umg/up.5i-some-values-here | 5i       | X           | XX

We recently updated to morpheus-cli 5.3.0.3 from 4.16 I think It was, and morpheus to 5.2.5 from the latest of the 4.2 branch

I haven't figured out a workaround yet by escaping the group name in morpheus-cli or quoting the name or anything

**Comment by jamesdickson6** [comment]
Ah, this was actually a bug with that command in general, not related to the name of the group having any certain characters.
Thanks for bringing it to our attention!
The issue is fixed in the next CLI release, version 5.3.1, which will be published in the next couple days.

**Comment by jamesdickson6** [comment]
@rjr162 Just in case it helps you out.

You can install the latest CLI code like this:

```shell
git clone https://github.com/gomorpheus/morpheus-cli.git
cd morpheus-cli
rake install
```

The [CLI gem](https://rubygems.org/gems/morpheus-cli) version 5.3.1 will be published in the next day or two and afterwards you can just use the standard update command:

```shell
gem install morpheus-cli
```

Let me know if I can assist.
